flink-batch-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/DerbyUtil.java

```diff
@@ -19,7 +19,6 @@
 
 import java.io.OutputStream;
 
-@SuppressWarnings("unused")
```

Comment:
since this class is only referenced indirectly it should still require the warning.